src/components/TodoItemLoader/TodoItemLoader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItemLoader: React.FC<Props> = ({ todo }) => {
+  return (
+    <div className="todo">
+      <label className="todo__status-label">
+        <input type="checkbox" className="todo__status" />
+      </label>
+
+      <span className="todo__title">{todo.title}</span>
+      <button type="button" className="todo__remove">×</button>
```

Comment:
Consider using a more descriptive label for the remove button, like `Remove Todo` or `Delete Todo`. 🗑️